modules/fedimint-ln/src/contracts/incoming.rs

```diff
@@ -13,6 +13,7 @@ pub struct IncomingContractOffer {
     pub amount: fedimint_api::Amount,
     pub hash: bitcoin_hashes::sha256::Hash,
     pub encrypted_preimage: EncryptedPreimage,
+    pub expiry_time: Option<u64>,
```

Comment:
We need a plan for what to do with this field. I'm ok with merging but the expiry should be enforced eventually (open issue or follow-up PR).